LICENSE-binary

```diff
@@ -303,3 +344,8 @@ org.apache.hbase.thirdparty:hbase-shaded-jersey
 The ISC License
 ---------------
 rimraf
+
+The Apache Software License, Version 2.0, LGPL License or MPL License
```

Comment:
When something is available under multiple licenses, you should select the license that is compatable with the ASF license/policy. See https://www.apache.org/legal/resolved.html#mutually-exclusive